src/ptdf_calculations.jl

```diff
@@ -439,15 +429,33 @@ Builds the PTDF matrix from a system. The return is a PTDF array indexed with th
 
 # Arguments
 - `sys::PSY.System`:
-        Power Systems system
+        PSY system for which the matrix is constructed
+
+# Keyword Arguments
+- `dist_slack::Vector{Float64}=Float64[]`:
+        vector of weights to be used as distributed slack bus.
+        The distributed slack vector has to be the same length as the number of buse
+- `reduce_radial_branches::Bool=false`:
```

Comment:
kwarg inconsistency again

Reply:
insonsistency due to the need to keep dist_slack and reduce_radial_branches explicit